seyik/app.js

```diff
@@ -111,6 +110,57 @@ app.post("/posts/register", async (req, res) => {
   res.status(201).json({ message: "postCreated" });
 });
 
+app.patch("/modify", async (req, res) => {
+  const { postingContent, user_id } = req.body;
+
+  await appDataSource.query(
+    `UPDATE posts
+    SET
+    content = ?
+    WHERE posts.user_id = ? 
+    `,
+    [postingContent, user_id]
+  );
+  const rows = await appDataSource.query(
+    `SELECT 
+      users.id as userId,
+      users.name as userName,
+      posts.id as postingId,
+      posts.title as postingTitle,
+      posts.content as postingContent
+      FROM users LEFT JOIN posts
+      ON posts.user_id = users.id
+  `
+  );
+  res.status(200).json({ data: rows });
+});
+
+app.delete("/postsdelete", async (req, res) => {
+  const { postId } = req.body;
+  await appDataSource.query(
+    `DELETE
+    FROM posts
+    WHERE posts.id = ${postId}
+    `
+  );
+  res.status(200).json({ message: "postingDeleted" });
```

Comment:
```suggestion
  res.status(204).send();
```
delete method에도 204라는 status code를 응답으로 더 많이 쓰고 잇습니다. 다만 204는 message로 no content를 의미하고 실제로 바디에 어떠한 데이터를 담아서 보내지 않으므로 send()만 해주면 됩니다.